packages/babel-plugin-transform-template-literals/src/index.js

```diff
@@ -83,19 +82,27 @@ export default declare((api, options) => {
           callExpressionInput.push(t.arrayExpression(raws));
         }
 
-        const init = t.callExpression(helperId, callExpressionInput);
-        annotateAsPure(init);
-        init._compact = true;
         scope.push({
           id: templateObject,
-          init,
           // This ensures that we don't fail if not using function expression helpers
           _blockHoist: 1.9,
```

Comment:
I think that we can remove this "private" parameter now, because it was needed to initialize the template in the right place (while the `var templateObject;` statement can be anywhere).

Reply:
I had no idea what this was good for that's why I left it. In this case I will remove it.

Do you know if there is anything written down that explains this kind of private parameters?